cmd/avrogen/main.go

```diff
@@ -86,7 +96,7 @@ func realMain(args []string, stdout, stderr io.Writer) int {
 	}
 	formatted, err := format.Source(buf.Bytes())
 	if err != nil {
-		_, _ = fmt.Fprintf(stderr, "Error: could not format code: %v\n", err)
+		_, _ = fmt.Fprintf(stderr, "Error: generated code \n%s\n could not be formatted: %v\n", buf.String(), err)
```

Comment:
This could potentially be very very large.

Reply:
I added that generated code detail because one does not really have access to what was generated & impossible to understand the error. I found it useful

Reply:
That seems reasonable, but long console output can be difficult to handle. What about adding a debug flag that allows it to be dumped into a file when this error is encountered?

Reply:
I made it dump the generated bytes to the same file where the data is expected to be even though it failed to format it => then one could check what would have been with their template